.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,48 @@
+name: Documentation issue
+description: README, package docs, or code comments are wrong, missing, or misleading.
+title: "[Docs] "
+labels: ["documentation"]
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)